pkgs/development/web/bun/default.nix

```
@@ -12,7 +12,7 @@
 }:
 
 stdenvNoCC.mkDerivation rec {
-  version = "0.6.3";
+  version = "bun-v0.6.4";
```

```suggestion
  version = "0.6.4";
```
`bun-v` is a prefix and should not be included in the version.

---

Sorry, my bad